test/Rentals.spec.ts

```diff
@@ -2467,17 +2541,29 @@ describe('Rentals', () => {
       await evmIncreaseTime(daysToSeconds(offerParams.rentalDays))
       await evmMine()
 
-      const setUpdateOperator = rentals.connect(tenant).setUpdateOperator(land.address, tokenId, newOperator)
+      const setUpdateOperator = rentals.connect(tenant).setUpdateOperator([land.address], [tokenId], [newOperator])
 
-      await expect(setUpdateOperator).to.be.revertedWith('Rentals#setUpdateOperator: CANNOT_UPDATE_OPERATOR')
+      await expect(setUpdateOperator).to.be.revertedWith('Rentals#setUpdateOperator: CANNOT_SET_UPDATE_OPERATOR')
     })
 
     it('should revert when the lessor tries to update the operator before the rental ends', async () => {
       await rentals.connect(lessor).acceptOffer({ ...offerParams, signature: await getOfferSignature(tenant, rentals, offerParams) })
 
-      const setUpdateOperator = rentals.connect(lessor).setUpdateOperator(land.address, tokenId, newOperator)
+      const setUpdateOperator = rentals.connect(lessor).setUpdateOperator([land.address], [tokenId], [newOperator])
+
+      await expect(setUpdateOperator).to.be.revertedWith('Rentals#setUpdateOperator: CANNOT_SET_UPDATE_OPERATOR')
+    })
 
-      await expect(setUpdateOperator).to.be.revertedWith('Rentals#setUpdateOperator: CANNOT_UPDATE_OPERATOR')
+    it('should revert when the provided param arrays have different lengths', async () => {
+      await expect(rentals.connect(lessor).setUpdateOperator([], [tokenId], [newOperator])).to.be.revertedWith(
+        'Rentals#setUpdateOperator: LENGTH_MISMATCH'
+      )
+      await expect(rentals.connect(lessor).setUpdateOperator([land.address], [], [newOperator])).to.be.revertedWith(
+        'Rentals#setUpdateOperator: LENGTH_MISMATCH'
+      )
+      await expect(rentals.connect(lessor).setUpdateOperator([land.address], [tokenId], [])).to.be.revertedWith(
+        'Rentals#setUpdateOperator: LENGTH_MISMATCH'
+      )
     })
```

Comment:
Should we need to add a test that reverts if you are using a valid LAND and another invalid? like what we did for the claim?

Reply:
Added